### PR TITLE
added get profile user api call

### PR DESCRIPTION
**Pull request by maxperei**

### CODE REVIEW

**Comment by ricbra** [comment]
Thank you for your contribution @maxperei 

Could you also add a unit test for the new method? You can check the tests folder to see how the other calls are tested.

File: resources/service.php [hunk 1]
```diff
@@ -232,6 +232,18 @@
...
+	        'uri' => 'users/{username}',
+	        'responseModel' => 'GetResponse',
+	        'parameters' => [
+		        'username' => [
+			        'type' => 'string',
+			        'location' => 'uri',
+			        'required' => true
+		        ],
+	        ],
+        ],
```
Comment (ricbra) [inline resources/service.php]: Could you change the indentation level so it matches the rest of the file?

**Comment by maxperei** [comment]
sorry to bother you but i got this when I type : `git pull-request maxperei/profile`
> Error creating pull request: Unprocessable Entity (HTTP 422)
> A pull request already exists for maxperei:profile.

File: resources/service.php (hunk 1 above)
Comment (maxperei) [reply resources/service.php]: oops, i should change indent settings, now it's good

**Comment by ricbra** [comment]
Looks good, thanks again @maxperei

**Pull request merged**